package.json

```diff
@@ -9,35 +9,118 @@
     "url": "https://github.com/doubtfire-lms/doubtfire-web.git"
   },
   "scripts": {
-    "start": "grunt"
+    "build": "npm-run-all clean:prod build:prod",
+    "build:dev": "webpack-dev-server --progress --config config/webpack/dev.webpack.conf.js --hot",
+    "build:prod": "webpack -p --progress --config config/webpack/prod.webpack.conf.js",
+    "build:serve": "npm-run-all build serve:prod",
+    "clean:dev": "rimraf dev/",
+    "clean:prod": "rimraf dist/",
+    "dev": "npm-run-all clean:dev --parallel verify:watch build:dev serve:dev",
+    "copyenv": "rm src/app/api/env.coffee && gulp copyenv",
+    "release": "npm-run-all verify test:coverage build test:browser",
+    "serve:dev": "echo Starting DEV server",
+    "serve:prod": "npm run serve:prod:http",
+    "serve:prod:http": "serve -C -p 8000 dist/",
+    "serve:prod:https": "serve-https -p 8000 -d dist/ --servername localhost",
+    "start": "npm run dev",
+    "test": "npm run test:unit",
+    "test:browser": "protractor config/testBrowser/protractor.conf.js",
+    "test:coverage": "npm run test:unit:once",
+    "test:unit": "karma start ./config/testUnit/karma.conf.js",
+    "test:unit:debug": "karma start ./config/testUnit/karma.conf.js --debug",
+    "test:unit:once": "karma start --singleRun=true ./config/testUnit/karma.conf.js",
+    "verify": "npm-run-all verify:coffee verify:css verify:js --silent",
+    "verify:css": "sass-lint -c config/verify/sasslint.yml --verbose  && echo ✅ verify:css success",
+    "verify:css:watch": "chokidar 'src/**/*.sass' 'src/**/*.scss' -c 'npm run verify:css' --initial --silent",
+    "verify:js": "eslint -c config/verify/.eslintrc \"src/**/*.js\" \"unitTest/**/*.js\" \"config/**/*.js\" && echo ✅ verify:js success",
+    "verify:js:watch": "chokidar 'src/**/*.js' 'unitTest/**/*.js' 'config/**/*.js' -c 'npm run verify:js' --initial --silent",
+    "verify:coffee": "gulp verify:coffee",
+    "verify:coffee:watch": "chokidar 'src/**/*.coffee' 'unitTest/**/*.coffee' 'config/**/*.coffee' -c 'npm run verify:coffee' --initial --silent",
+    "verify:watch": "npm-run-all --parallel verify:coffee:watch verify:css:watch verify:js:watch --silent",
+    "postinstall": "./node_modules/protractor/bin/webdriver-manager update"
+  },
+  "dependencies": {
+    "angular": "1.3.19",
+    "angular-bootstrap": "~0.12.0",
```

Comment:
Can we silence those warnings? A bit annoying having hundreds of warnings when we could resolve them.

> On 7 Aug 2016, at 12:46 PM, John <notifications@github.com> wrote:
> 
> In package.json <https://github.com/doubtfire-lms/doubtfire-web/pull/34#discussion_r73794894>:
> 
> > +    "test:unit": "karma start ./config/testUnit/karma.conf.js",
> > +    "test:unit:debug": "karma start ./config/testUnit/karma.conf.js --debug",
> > +    "test:unit:once": "karma start --singleRun=true ./config/testUnit/karma.conf.js",
> > +    "verify": "npm-run-all verify:coffee verify:css verify:js --silent",
> > +    "verify:css": "sass-lint -c config/verify/sasslint.yml --verbose  && echo ✅ verify:css success",
> > +    "verify:css:watch": "chokidar 'src/**/*.sass' 'src/**/*.scss' -c 'npm run verify:css' --initial --silent",
> > +    "verify:js": "eslint -c config/verify/.eslintrc \"src/**/*.js\" \"unitTest/**/*.js\" \"config/**/*.js\" && echo ✅ verify:js success",
> > +    "verify:js:watch": "chokidar 'src/**/*.js' 'unitTest/**/*.js' 'config/**/*.js' -c 'npm run verify:js' --initial --silent",
> > +    "verify:coffee": "gulp verify:coffee",
> > +    "verify:coffee:watch": "chokidar 'src/**/*.coffee' 'unitTest/**/*.coffee' 'config/**/*.coffee' -c 'npm run verify:coffee' --initial --silent",
> > +    "verify:watch": "npm-run-all --parallel verify:coffee:watch verify:css:watch verify:js:watch --silent",
> > +    "postinstall": "./node_modules/protractor/bin/webdriver-manager update"
> > +  },
> > +  "dependencies": {
> > +    "angular": "1.3.19",
> > +    "angular-bootstrap": "~0.12.0",
> Actually that's a mistake, angular-bootstrap shouldn't be there, the relevant one is angular-ui-bootstrap.
> 
> I did bump the version of angular-ui-bootstrap but it is not the latest (0.12.0 -> 0.14.3).
> 
> as a result there are warnings that where possible uib-* should be used but it does not cause any errors
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub <https://github.com/doubtfire-lms/doubtfire-web/pull/34/files/2465065b864b772d6f34e8733f215c21603d996b#r73794894>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABXUQGVjsmTv18nZDnYPuZc0lI9o4Me3ks5qdUcCgaJpZM4JeWks>.
>